docs/architecture/003-namespaced-keywords.md

```diff
@@ -53,3 +56,42 @@ Pros of namespaced keys:
 Cons:
 
 - verbose (mitigated by using short namespaces)
+
+# Amendment: Same data for same key
+
+Date: 2019-11-05
+Authors: @opqdonut @Macroz
+
+When using namespaced keys, it's useful to keep the structure of the
+corresponding values fixed. That is, don't mix forms like this:
+
+```clojure
+{:blacklist/user "bob"
+ :blacklist/resource :123"}
+{:blacklist/user {:userid "bob" :name "Bob"}
+ :blacklist/resource {:resource/ext-id "urn:123" :resource/id 3}}
+```
+
+Instead use partial maps like this:
+
+```clojure
+{:blacklist/user {:userid "bob"}
+ :blacklist/resource {:resource/ext-id "urn:123"}}
+```
+
+Or reuse known keys like this:
```

Comment:
I didn't understand straight away what this means. Could it be clarified a bit?

Reply:
Tried to clarify